build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -79,12 +83,19 @@
       <property name="logLoadErrors" value="true"/>
       <property name="suppressLoadErrors" value="true"/>
     </module>
+<!--    The below *should* work but is currently broken: https://github.com/checkstyle/checkstyle/issues/6192-->
+<!--    <module name="JavadocParagraph">-->
+<!--      &lt;!&ndash; This rule ensures a new line before parameters. &ndash;&gt;-->
+<!--      &lt;!&ndash; And this means <p> elements can appear on a line of their own. &ndash;&gt;-->
+<!--      <property name="allowNewlineParagraph" value="true"/>-->
+<!--    </module>-->
+    <module name="JavadocTagContinuationIndentation">
+      <property name="offset" value="1"/>
```

Comment:
I would say this should be 2 given our standard indent, but not sure what everyone has been doing

Reply:
There seems to be a rough split whereby the OSS components use 2 and the private ones use 1.

Reply:
This is down to the fact that Intellij autoformatter doesn't allow javadoc tag spacing to be customised, so our OSS is done by hand and our internal is left to the autoformat